scrapers/Noodledude.yml

```diff
@@ -0,0 +1,36 @@
+name: "NoodleDude"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - noodledude.io
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //meta[@property='og:title']/@content
+        postProcess:
+          - replace:
+            - regex: (.*) \| NoodleDude PMVs$
+              with: $1
+      Details: //meta[@name='description']/@content
```

Comment:
The description from the meta ends up having the  related tags text at  the end (and we already have the tags anyway)
Do we want to postprocess and remove them?
Do we care for the extra info as voice over and music?
Something like
```yaml
      Details:
        selector: //div[@class="description padding-medium"]/div[@class="rich-text-block-2 text-rich-text w-richtext"]//p
        concat: "\n\n"
```
seemed to work, didnt test with all pmvs though

Reply:
Info about voice / music is useful to have for PMVs yeah.
Your query didn't work for all links, I had to rework it a bit but now it properly grabbed the description for all my test cases.